### PR TITLE
feat: rendered the question on ui check correct question

### PR DESCRIPTION
**Pull request by Rockyraven**
feat:
rendered the question on ui, checked correct question
updated score on selection of the option

### CODE REVIEW

File: src/API/Category1.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                        options: shuffle([...item.incorrect_answers, item.correct_answer]),
+                        answer: item.correct_answer
+                    }
+                )));
+            })
+            .catch(err => console.error(err))
+    }, []);
+
+
+    // useEffect(() => {
```
Comment (arjun-computer-geek) [inline src/API/Category1.js]: Remove commented code

File: src/API/Category1.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    <h3 dangerouslySetInnerHTML={{ __html: quiz[number].question }}></h3>
+
+                    <div className='ans-Option'>
+                        {quiz[number].options.map((item, index) => (
+                            <button key={index} dangerouslySetInnerHTML={{ __html: item }} onClick={pickAnswer}></button>
+                        ))}
+                    </div>
+                </>
+            }
+            {/* {
```
Comment (arjun-computer-geek) [inline src/API/Category1.js]: Here too

File: src/API/Category1.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                number === 5 && <GameOver pts={pts} />
+            } */}
+
```
Comment (Shubhamkr2610) [inline src/API/Category1.js]: Suggestion: remove unused code while pushing the code

File: src/component/card/QuizCard.js [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
 
-export const QuizCard = ({heading, about, Image}) => {
+export const QuizCard = ({title, image,_id}) => {
   return (
     <div className='quiz-card-container'>
-        <img src={category1} alt="" />
-        <p className='card-heading'>{heading}</p>
-        <p className="card-description">{about}</p>
-        <a href="#" className='play-button'>Play Now</a>
+        <img src={image} alt="" />
```
Comment (arjun-computer-geek) [inline src/component/card/QuizCard.js]: Give the alt text

File: src/component/categories/CategoryQuiz.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { categories } from 'db/category-db'
+import { QuizCard } from 'component/card/QuizCard'
+
+export const CategoryQuiz = () => {
+  return (
+  <>
+      {categories.map(category => <QuizCard title={category.name} _id={category._id} image={category.image}/>)}
```
Comment (arjun-computer-geek) [inline src/component/categories/CategoryQuiz.jsx]: Always use key prop while mapping

File: src/pages/quizpage/quizpage.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
+*{
+    margin: 0px;
+    padding: 0px;
+   
+}
```
Comment (Shubhamkr2610) [inline src/pages/quizpage/quizpage.css:5]: remove extra spacing

File: src/context/questionContext.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    const [ questionCategoryList, setQuestionCategoryList ] = useState();
+
+    const fetchQuestion = (questionID) => {
+        let questionCategoryExit = categories.find((item) => item._id === questionID)
+
+        fetch(questionCategoryExit.link)
+        .then(response => response.json() )
+        .then( data => setQuestionCategoryList(data.results))
+    }
+    // console.log(questionCategoryList)
```
Comment (arjun-computer-geek) [inline src/context/questionContext.js]: Remove commented code

File: src/pages/quizpage/quizpage.css [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+.correct {
+    background-color: rgb(7, 207, 0);
+    color: white;
+    box-shadow: 0 0 1px black;
+  }
+  .wrong {
+    background-color: rgb(233, 0, 0);
+    color: white;
```
Comment (Shubhamkr2610) [inline src/pages/quizpage/quizpage.css:57]: maintain consistency with naming of color

**Review by Shubhamkr2610 — APPROVED** [review]
gave some suggestions , after fixing , You are good to go .

**Review by arjun-computer-geek — APPROVED** [review]
Do some suggested changes. Else looks good

**Pull request merged**